code/modules/events/immovable_rod.dm

```diff
@@ -86,14 +86,21 @@ In my current plan for it, 'solid' will be defined as anything with density == 1
 				"<span class ='danger'>You hear a CLANG!</span>")
 			H.adjustBruteLoss(160)
 		if(clong.density || prob(10))
-			clong.ex_act(2)
+			clong.ex_act(EXPLODE_HEAVY)
 
 /obj/effect/immovablerod/event
 	var/tiles_moved = 0
+	// The base chance to "damage" the floor when passing. This is not guaranteed to cause a full on hull breach.
+	// Chance to expose the tile to space is like 60% of this value.
+	var/floor_rip_chance = 40
```

Comment:
If this isn't intended to change based on rod type or through some other event then it may be a better idea just to slap this in a #define

Reply:
its changed for wizard rods.

Reply:
I'm strongly opposed to `#define`s on the grounds that they pollute the global namespace.
I would not mind hardcoding the value, but i still feel the variable is nicer.

Changing this for adminbus is very possible, but requires a bit of a finesse, since you need to VV an already flying rod - you cannot tweak upcoming event parameters, since events don't support having tweakable parameters (currently).

And as for wizard rod.. I.. really don't have any opinion on wizards...
I could move the `var`, and the floor handling up one level, and have something like
```dm
/obj/effect/immovablerod
  var/floor_rip_chance = 0
// base type
/obj/effect/immovablerod/Move()
	var/atom/oldloc = loc
	. = ..()
	if(prob(floor_rip_chance))
		var/turf/simulated/floor/T = get_turf(oldloc)
		if(istype(T))
			T.ex_act(EXPLODE_HEAVY)
// event specifically
/obj/effect/immovablerod/event
	floor_rip_chance = 40
// event Move proc, exactly as it is today.
/obj/effect/immovablerod/event/Move()
	var/atom/oldloc = loc
	. = ..()
	tiles_moved++
	if(get_dist(oldloc, loc) > 2 && tiles_moved > 10) // We went on a journey, commit sudoku
		qdel(src)
```
But I don't think that's better. It's more "enterprisey" though...

So I'll leave the code as it is for now, unless/until someone has strong opinions.